### PR TITLE
Pending BR strings

### PR DESCRIPTION
**Pull request by vitu**
My apologies for being late to the party. Hope you're doing great!

I seem to have found a string not yet internationalized: “Set as fallback folder” (button to select a folder in a dialog). 
• 2023-09-09 às 19 08 16

¡Salud!

### CODE REVIEW

**Comment by aonez** [comment]
Thanks a lot Victor! Just pushed that string to the repo 🙌🏼

File: Translations/pt-BR.lproj/Localizable.strings
Comment (aonez) [inline Translations/pt-BR.lproj/Localizable.strings:1]: Ops, that one is empty...

**Comment by aonez** [comment]
@vitu  sadly both Localizable and InfoPlist files are corrupt or unusable. I've checked your fork and they were uploaded like that... Can't see if you've changed anything there.

**Comment by vitu** [comment]
My bad… as you're aware, things got messy here. Thanks for helping me find my way back, though! For this new string, could you please use “Definir como Pasta Alternativa”? Salud!

**Comment by aonez** [comment]
Done! Thanks again 🙌🏼

**Pull request merged**